### ISSUE TITLE
Labels Only Placed At Whole and Half Phases

### ISSUE THREAD

**Issue by MutantPlatypus** [wallento/wavedrompy#11]
Edge labels are placed where an edge would be for a whole and half phase, instead of being placed where the edge actually is after phase shifting.

For the below JSON only labels "a", "f", and "k" are placed correctly:
```
{ "signal": [
  { "name": "0",   "wave": "lh", "node": ".a", "phase":0},
  { "name": "0.1", "wave": "lh", "node": ".b", "phase":-0.1},
  { "name": "0.2", "wave": "lh", "node": ".c", "phase":-0.2},
  { "name": "0.3", "wave": "lh", "node": ".d", "phase":-0.3},
  { "name": "0.4", "wave": "lh", "node": ".e", "phase":-0.4},
  { "name": "0.5", "wave": "lh", "node": ".f", "phase":-0.5},
  { "name": "0.6", "wave": "lh", "node": ".g", "phase":-0.6},
  { "name": "0.7", "wave": "lh", "node": ".h", "phase":-0.7},
  { "name": "0.8", "wave": "lh", "node": ".i", "phase":-0.8},
  { "name": "0.9", "wave": "lh", "node": ".j", "phase":-0.9},
  { "name": "1",   "wave": "lh", "node": ".k", "phase":-1}
  ],
  "edge": [
    "a-b",
    "b-c",
    "c-d",
    "d-e",
    "e-f",
    "f-g",
    "g-h",
    "h-i",
    "i-j",
    "j-k"
  ]
}
```
WaveDromPy:
![image](https://user-images.githubusercontent.com/1389865/63895183-68280a80-c9bc-11e9-93c1-7690e4955a31.png)

WaveDrom:
![image](https://user-images.githubusercontent.com/1389865/63895217-7e35cb00-c9bc-11e9-9ce8-f110b87ca467.png)

**Comment by MutantPlatypus** [comment]
Edited JSON to use "lh" wave instead of "lp", use better wave names (name=phase), and removed unnecessary and confusing line labels.

**Comment by wallento** [comment]
Thanks for reporting. There is an undocumented feature in wavedrompy which allows a better alternative to standard labels, but this should definitely also work. I should be able to fix it until end of the week.

**Comment by wallento** [comment]
Fixed in d570c47. Thanks for reporting!
